code/controllers/subsystem/daynightcycle.dm

```diff
@@ -0,0 +1,77 @@
+var/datum/subsystem/daynightcycle/SSDayNight
+
+#define TOD_MORNING 	1
+#define TOD_SUNRISE 	2
+#define TOD_DAYTIME 	3
+#define TOD_AFTERNOON 	4
+#define TOD_SUNSET 		5
+#define TOD_NIGHTTIME 	6
+
+/datum/subsystem/daynightcycle
+	name          = "Day Night Cycle"
+	init_order    = SS_INIT_DAYNIGHT
+	display_order = SS_DISPLAY_DAYNIGHT
+	priority      = SS_PRIORITY_DAYNIGHT
+	wait          = 20 MINUTES
+	flags         = SS_BACKGROUND|SS_FIRE_IN_LOBBY
+
+	var/current_color = "#4d3d66"
+	var/timeOfDay = TOD_MORNING
+	var/asspower = 10
+	var/assrange = 1
+
+/datum/subsystem/daynightcycle/New()
+	NEW_SS_GLOBAL(SSDayNight)
+
+/datum/subsystem/daynightcycle/Initialize()
+	var/list/timestwopick = list(TOD_MORNING,
+								TOD_SUNRISE,
+								TOD_DAYTIME,
+								TOD_AFTERNOON,
+								TOD_SUNSET,
+								TOD_NIGHTTIME)
+	timeOfDay = pick(timestwopick)
```

Comment:
For consistency I might seriously consider always starting at noon or dawn - but this one is pure opinion.

Reply:
Removed the RNG Time of day pick, it will always start at Afternoon now